rllib/examples/rl_modules/classes/autoregressive_actions_rlm.py

```diff
@@ -49,6 +49,7 @@ def setup(self):
 
         # Build the encoder.
         self.encoder = self.catalog.build_encoder(framework=self.framework)
+        self.encoder = self.catalog.build_encoder(framework=self.framework)
```

Comment:
why duplicate this line?

Reply:
This is probably from the conflict in a merge.